000-vip-init.php

```diff
@@ -198,6 +198,41 @@
 	}
 }
 
+// Set WordPress >5.5 environment name
+if ( defined( 'VIP_GO_APP_ENVIRONMENT' ) && ! defined( 'WP_ENVIRONMENT_TYPE' ) ) {
```

Comment:
Can we make this code more testable, maybe by moving it into one or two functions?

Reply:
I thought about declaring a function, but there weren't any other in this file. This leads me to think that maybe this is not the best place to put this. Any other suggestion? Would there be a better place to initialize this, or should I keep it in this file inside a couple of functions?

Reply:
> but there weren't any other in this file

I guess the tricky bit is that it's still hard to test even with a function, b/c our test bootstrap file includes 000-vip-init.php, which will define the constants and change the behavior of tests. We should rethink how that works long term...I guess for now it won't make much of a difference in practice.